pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -111,6 +112,37 @@ protected List<Object[]> queryRunner(String sql) {
         queryPlan.getQueryResultFields(), queryPlan.getQueryStageMap().get(0).getDataSchema()).getRows();
   }
 
+  protected List<Object[]> queryRunner(String sql, ExecutionStatsAggregator executionStatsAggregator) {
```

Comment:
simply make the normal queryRunner API call this one with a null executionStatsAggregator?

the only difference is the last step when reduceMailboxReceive yes?

Reply:
Done.